packages/fx-core/templates/plugins/resource/apim/bicep/config.template.bicep

```diff
@@ -1,4 +1,4 @@
-module teamsFxApimConfig '{{fx-resource-apim.Configuration.apim.path}}' = {
```

Comment:
there may be no need to compile `config.template.bicep` in resource plugin, if all the placeholders need to be expanded in solution plugin. In this way, we do not need to add `\` here.